sphinx_gallery/gen_rst.py

```diff
@@ -284,34 +287,19 @@ def get_md5sum(src_file):
     return src_md5
 
 
-def check_md5sum_change(src_file):
-    """Returns True if src_file has a different md5sum"""
+def md5sum_is_current(src_file):
```

Comment:
It could well be that the .md5 is written in doc/auto_examples but when you read it you expect it to be in examples. I think the former would be preferrable to be honest in order to keep the git repo (and the git status output) cleaner.